### DISCUSSION TITLE
Having trouble Installing the map card, specifying the resource url

### DISCUSSION THREAD

**Discussion by VrtlBrown** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#712] [Q&A]
### Checklist

- [X] I have updated the card to the latest version available
- [X] I have cleared the cache of my browser
- [X] I have checked if the problem is already reported

### The problem

Hi I've installed the card via HACS, the full path of the files I installed is:
/volume1/docker/homeassistant/config/www/community/lovelace-xiaomi-vacuum-map-card/xiaomi-vacuum-map-card.js

But no matter what I insert into the resource url field it wont load the card on the dashboard. 

What should I use for the url? 

### What version of a card has described problem?

v2.2.2

### What vacuum model do you have problems with?

S6 maxV

### Which integration do you use to control your vacuum (link)?

https://www.home-assistant.io/integrations/xiaomi_miio/

### What browser (browsers/apps) does have this problem?

Safari

### What version of Home Assistant do you use?

2024.1.6

### What type of installation are you running?

Home Assistant Container

### Card's configuration

```yaml
Haven't got this far yet
```

**Comment by sca075** [comment]
@VrtlBrown ,The path should be something like: 
```
/config/www/community/custom_lovelace/xiaomi-vacuum-map-card.js
```
the same goes in the resources but there _www_ need to be replaced with **local**
```
/local/custom_lovelace/xiaomi-vacuum-map-card.js
```
hope this helps

**Comment by PiotrMachowski** [comment]
If you installed it with HACS then it should automatically add it to resources. If not, it should be something like this:
```
/hacsfiles/lovelace-xiaomi-vacuum-map-card/xiaomi-vacuum-map-card.js?hacstag=331701152204
```
